cumulus/parachains/integration-tests/emulated/assets/asset-hub-westend/src/tests/teleport.rs

```diff
@@ -361,3 +361,15 @@ fn teleport_native_assets_from_relay_to_system_para_works() {
 // 	// Receiver's balance does not change
 // 	assert_eq!(receiver_balance_after, receiver_balance_before);
 // }
+
+#[test]
+fn teleport_to_other_system_parachains_works() {
+	let amount = ASSET_HUB_WESTEND_ED * 100;
+	let native_asset: VersionedMultiAssets = (Parent, amount).into();
+
+	test_parachain_is_trusted_teleporter!(
+		AssetHubWestend,          // Origin
+		vec![CollectivesWestend], // Destinations
```

Comment:
Westend has a Bridge Hub, do we not have a test case for it?

Reply:
Added